api/external/infra_proxy/response/environments.proto

```diff
@@ -16,16 +16,16 @@ message EnvironmentListItem {
 message Environment {
     // Environment name.
     string name = 1;
-    // Object type.
+    // Chef object type.
     string chef_type = 2;
-    // Node description.
+    // Environment description.
     string description = 3;
-    // Class name.
-    string json_class = 5;
-    // Versioned cookbooks list.
-    repeated string cookbook_versions = 6;
+    // Environment JSON class.
+    string json_class = 4;
+    // Environment versined cookbooks constraints.
+    map<string, string> cookbook_versions = 5;
```

Comment:
Changed the data type of `cookbook_versions` from array to map causes data load error in the environment get API

Reply:
This might cause a break in the existing environment detail UI.